### PR TITLE
Feature/votingfeedback now works

### PR DESCRIPTION
**Pull request by ZacharyBys**

### CODE REVIEW

File: frontend/src/components/CaptionVotes.js [hunk 1]
```diff
@@ -2,12 +2,28 @@ import React from 'react';
 import '../styles/CaptionVotes.css';
 
 const CaptionVotes = props => {
+    var upVote = <li className="vote-ticker" id="+" onClick={props.voteHandler}>+</li>
+    var downVote = <li className="vote-ticker" id="-" onClick={props.voteHandler}>-</li>
```
Comment (tehblasian) [inline frontend/src/components/CaptionVotes.js]: use `let` not `var`

File: frontend/src/styles/CaptionVotes.css [hunk 2]
```diff
@@ -26,4 +26,14 @@
 
 .vote-ticker:hover{
```
Comment (tehblasian) [inline frontend/src/styles/CaptionVotes.css]: Did you forget to change the class name here?

File: frontend/src/components/CaptionVotes.js [hunk 3]
```diff
@@ -2,12 +2,28 @@ import React from 'react';
...
+
+    switch (props.vote_value) {      
+        case 1:
+            upVote = <li className="vote-up" id="+" onClick={props.voteHandler}>+</li>
+            break;
+
```
Comment (tehblasian) [inline frontend/src/components/CaptionVotes.js]: remove the whitespace while you're at it

**Review by tehblasian — CHANGES_REQUESTED** [review]
See comments

File: frontend/src/styles/CaptionVotes.css (hunk 2 above)
Comment (ZacharyBys) [reply frontend/src/styles/CaptionVotes.css]: Good catch, thanks

**Review by tehblasian — APPROVED** [review]
Good, this will make remembering user votes great again.

**Pull request merged**